plugins/default/lib/openshift.py

```diff
@@ -5,18 +5,31 @@
 def oc(*args):
     return check_output(("oc",) + args)
 
+
 def _get_running_pod_names(oc, project):
     # Manually filter Running pods because of a bug in `oc get`, see:
     # https://github.com/kubernetes/kubernetes/issues/29115
-    #return oc("-n", project, "get", "pods", "--show-all=false", "-o", "name")
-    pods = json.loads(oc("-n", project, "get", "pods", "--show-all=false", "-o", "json"))
-    return [p["metadata"]["name"] for p in pods if p["status"]["phase"] == "Running"]
+    # return oc("-n", project, "get", "pods", "--show-all=false", "-o", "name")
+    pods = json.loads(oc("-n", project, "get", "pods","--show-all=false", "-o", "json"))
+    return [p["metadata"]["name"] for p in pods["items"] if p["status"]["phase"] == "Running"]
```

Comment:
I think my previous comment was unclear.

I meant:

```python
pods = json.loads(oc("-n", project, "get", "pods", "--show-all=false", "-o", "json"))["items"]
return [p["metadata"]["name"] for p in pods if p["status"]["phase"] == "Running"]
```

If we had tests, I possibly wouldn't have introduced this bug. I see the tests are empty for this function:
https://github.com/feedhenry/nagios-container/pull/7/files#diff-143e4678f975ff3a05e70c1a679a9fd9R7 :cry: